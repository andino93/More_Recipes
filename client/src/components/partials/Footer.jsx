import React from 'react';

/** 
 * @export
 * @class FooterComponent
 * @extends {React.Component}
 */
export default class FooterComponent extends React.Component {
  /**
     * 
     * 
     * @returns {XML} XML/JSX
     * @memberof FooterComponent
     */
  render() {
    return (
      <footer className="p-10 well well-sm m-0 shadow-lite">
        <div className="text-center text-muted">
          <p className="bold">&copy; 2017 More Recipes</p>
        </div>
      </footer>
    );
  }
}

