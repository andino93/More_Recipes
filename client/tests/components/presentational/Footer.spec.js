import React from 'react';
import expect from 'expect';
import Adapter from 'enzyme-adapter-react-16';
import toJson from 'enzyme-to-json';
import { shallow, configure } from 'enzyme';
import Footer from '../../../src/components/partials/Footer.jsx';

configure({ adapter: new Adapter() });

describe('<Footer>', () => {
  let shallowComponent;
  beforeEach(() => {
    shallowComponent = shallow(<Footer/>);
  });
  it('renders <Footer> component', () => {
    const tree = toJson(shallowComponent);
    expect(tree).toMatchSnapshot();
  });
});
