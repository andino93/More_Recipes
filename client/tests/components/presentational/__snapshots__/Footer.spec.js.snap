// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer> renders <Footer> component 1`] = `
<footer
  className="p-10 well well-sm m-0 shadow-lite"
>
  <div
    className="text-center text-muted"
  >
    <p
      className="bold"
    >
      Â© 2017 More Recipes
    </p>
  </div>
</footer>
`;
